frontend/public/components/environment.jsx

```diff
@@ -285,371 +286,384 @@ const stateToProps = ({ k8s, UI }, { obj }) => ({
-      }
-
-      const envVar = currentEnvVars.getEnvVarByTypeAndIndex(containerType, containerIndex);
-
-      const containerDropdown = currentEnvVars.isContainerArray ? (
-        <ContainerDropdown
-          currentKey={rawEnvData[containerType][containerIndex].name}
-          containers={getContainersObjectForDropdown(rawEnvData.containers)}
-          initContainers={getContainersObjectForDropdown(rawEnvData.initContainers)}
-          onChange={this.selectContainer}
-        />
-      ) : null;
-
-      const owners = _.get(obj.metadata, 'ownerReferences', []).map((o, i) => (
-        <ResourceLink
-          key={i}
-          kind={referenceForOwnerRef(o)}
-          name={o.name}
-          namespace={obj.metadata.namespace}
-          title={o.uid}
-          inline
-        />
-      ));
-      const containerVars = (
-        <>
-          {readOnly && !_.isEmpty(owners) && (
-            <div className="co-toolbar__group co-toolbar__group--left">
-              <Alert
-                isInline
-                className="co-alert col-md-11 col-xs-10"
-                variant="info"
-                title="Environment variables set from parent"
-              >
-                View environment for resource {owners.length > 1 ? <>owners: {owners}</> : owners}
-              </Alert>
-            </div>
-          )}
-          {currentEnvVars.isContainerArray && (
-            <div className="co-toolbar__group co-toolbar__group--left">
-              <div className="co-toolbar__item">
-                {containerType === 'containers' ? 'Container:' : 'Init Container:'}
-              </div>
-              <div className="co-toolbar__item">{containerDropdown}</div>
+    const envVar = currentEnvVars.getEnvVarByTypeAndIndex(containerType, containerIndex);
+
+    const containerDropdown = currentEnvVars.isContainerArray ? (
+      <ContainerDropdown
+        currentKey={rawEnvData[containerType][containerIndex].name}
+        containers={getContainersObjectForDropdown(rawEnvData.containers)}
+        initContainers={getContainersObjectForDropdown(rawEnvData.initContainers)}
+        onChange={this.selectContainer}
+      />
+    ) : null;
+
+    const owners = _.get(obj.metadata, 'ownerReferences', []).map((o, i) => (
+      <ResourceLink
+        key={i}
+        kind={referenceForOwnerRef(o)}
+        name={o.name}
+        namespace={obj.metadata.namespace}
+        title={o.uid}
+        inline
+      />
+    ));
+    const containerVars = (
+      <>
+        {readOnly && !_.isEmpty(owners) && (
+          <div className="co-toolbar__group co-toolbar__group--left">
+            <Alert
+              isInline
+              className="co-alert col-md-11 col-xs-10"
+              variant="info"
+              title={t('environment~Environment variables set from parent')}
+            >
+              {t('environment~View environment for resource')}{' '}
+              {owners.length > 1 ? <>owners: {owners}</> : owners}
```

Comment:
"owners" should be localized

Reply:
It looks like this comment is still relevant?

Reply:
Whoops, sorry. I remember making this change, but must not have committed it.